bpf/tests/ratelimit.c

```diff
@@ -23,7 +23,12 @@ CHECK("xdp", "ratelimit") int test_ratelimit(void)
 		.topup_interval_ns = NSEC_PER_SEC,
 	};
 	struct ratelimit_key key = {
-		.netdev_idx = 1,
+		.usage = 0,
```

Comment:
Should we set
```suggestion
		.usage = RATELIMIT_USAGE_ICMPV6,
```
For consistency and since `icmpv6.netdev_idx` is set in the key?

Reply:
Sure.